app/controllers/fullcalendar_engine/events_controller.rb

```diff
@@ -107,7 +108,7 @@ def load_event
     end
 
     def event_params
-      params.require(:event).permit('title', 'description', 'starttime', 'endtime', 'all_day', 'period', 'frequency', 'commit_button')
+      params.require(:event).permit('title', 'description', 'starttime', 'endtime', 'all_day', 'period', 'frequency', 'commit_button', 'color')
```

Comment:
can we please symbolize all the keys?
I know this is left out from our side but you can please make these keys as symbols.